view/frontend/templates/breadcheckout/cartview.phtml

```diff
@@ -109,25 +132,43 @@
                     shippingAddress.block_key        = '<?php /* @noEscape */ echo $block->getBlockCode(); ?>';
                     shippingAddress.additional_data  = <?php /* @noEscape */ echo  $block->jsonEncode($block->setAdditionalData()); ?>;
 
+                    var shippingAddressEstimationUrl = '<?php /* @noEscape */ echo $block->getShippingAddressEstimationUrl(); ?>';
+                    var errorInfo;
+
                     $.ajax({
-                        url: '<?php /* @noEscape */ echo $block->getShippingAddressEstimationUrl(); ?>',
+                        url: shippingAddressEstimationUrl,
                         data: shippingAddress,
                         type: 'post'
                     }).done(function (response) {
                         try {
+                            errorInfo = {
+                                response: response,
+                                shipping_info: shippingAddress,
+                            };
                             if (typeof response == 'object') {
```

Comment:
Just a small thing - in a few places, you use `response !== null && typeof response === 'object'` which seems like a more robust check. Up to you if you want to change these though!

Reply:
yeah I'll change that everywhere else to be more consistent across the plugin